src/index.js

```diff
@@ -1 +1,21 @@
 import './style.css';
+import newGame from './modules/newGame.js';
+import postUserScore from './modules/formSubmit.js';
+import renderUserScores from './modules/renderScore.js';
+
+const formSubmitBtn = document.querySelector('#form-submit-btn');
+const refreshButton = document.getElementById('refresh-btn');
+
+window.addEventListener('load', () => {
+  newGame();
+});
+
+formSubmitBtn.addEventListener('click', (event) => {
+  event.preventDefault();
+  postUserScore();
+});
+
+refreshButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  renderUserScores();
+});
```

Comment:
- [ ] Clicking the refresh button renders the game scores twice, Please make sure that the old scores data get removed before getting the new data when clicking refresh.

Reply:
This is a bug and I would take a look and fix it. Thanks for the review